src/picongpu/include/particles/creation/creation.hpp

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright 2015-2016 Heiko Burau
```

Comment:
This file contains a lot of code from the file `ionization.hpp` therefore please keep the original authors.